src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  function getSum(a) {
+    return function(b) {
+      counter++;
+
+      switch (true) {
+        case (counter >= 3 && counter % 2 === 0):
+          return 'Bzzz... Error!';
+
+        default:
+          return a + b;
+      }
```

Comment:
it would be better to not overcomplicate and use simple `if ... else` construction